examples/tina-cloud-starter/components/tina-wrapper.tsx

```diff
@@ -87,4 +84,82 @@ const Inner = (props) => {
   )
 }
 
+import { Cloudinary } from 'cloudinary-core'
+import {
+  Media,
+  MediaList,
+  MediaListOptions,
+  MediaStore,
+  MediaUploadOptions,
+} from 'tinacms'
+
+export class CloudinaryMediaStore2 implements MediaStore {
```

Comment:
Is this code needed?

Reply:
It is not, was some debugging code I forgot to remove. Just pushed up a removal.